test/karma_test.go

```diff
@@ -0,0 +1,178 @@
+package test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+
+	testcluster "github.com/mesosphere/ksphere-testing-framework/pkg/cluster"
+	testharness "github.com/mesosphere/ksphere-testing-framework/pkg/harness"
+	networkutils "github.com/mesosphere/ksphere-testing-framework/pkg/utils/networking"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	karmaAlertsPath    = "/ops/portal/kommander/monitoring/karma/alerts.json"
+	karmaHealthyStatus = "success"
+	karmaPodPrefix     = "kommander-kubeaddons-karma-"
+	karmaPort          = "8080"
+	ns                 = "kommander"
+)
+
+// -----------------------------------------------------------------------------
+// Karma - Tests
```

Comment:
@juliangieseke we have _some_ karma and thanos tests now :)

Reply:
Not in this repo? Thir PR doesnt add/change anything, its just a reminder to “release” a beta of kommander.